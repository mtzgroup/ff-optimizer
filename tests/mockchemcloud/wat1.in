method  uhf
charge 1
spinmult 2
coordinates wat1.xyz
resp yes
run gradient
basis 6-31g*
threall 1e-15
convthre 1e-6
precision double
