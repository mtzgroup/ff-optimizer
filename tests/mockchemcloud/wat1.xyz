3
converted from 3.pdb
O	0.684	-1.717	-2.085
H	0.397	-1.119	-1.446
H	1.178	-2.183	-2.804
