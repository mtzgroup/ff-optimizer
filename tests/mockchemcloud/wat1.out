| Startfile from command line: wat1.in


|       ***********************************************************
|       *                 TeraChem v1.9-2025.01-dev               *
|       *                   Development Version                   *
|       *           Chemistry at the Speed of Graphics!           *
|       ***********************************************************
|       * This program may only be used in connection with        *
|       * a valid license from PetaChem, LLC. Use of this program *
|       * or results thereof indicates acceptance of all terms    *
|       * and conditions stated in the license and that a valid   *
|       * license agreement between the user and PetaChem, LLC    *
|       * exists. PetaChem, LLC does not warrant the correctness  *
|       * of results or their suitability for any purpose.        *
|       * Please email bugs, suggestions, and comments to         *
|       *                  help@petachem.com                      *
|       *                                                         *
|       ***********************************************************


|       ***********************************************************
|       *  Compiled by root         Wed Feb  5 07:16:34 PM PST 2025  *
|       *  Supported architecture SMs: 52 61 70 80 90             *
|       *  Build cuda_12.2.r12.2/compiler.33053471_0              *
|       *  Git Version:                               *
|       ***********************************************************


| Job started   Wed Feb 19 10:43:33 2025
| On fire-09-07 (available memory: 50742 MB)

| ######################################### RUNTIME INFO ##########################################
| terachem  wat1.in 

|  NVRM version: NVIDIA UNIX x86_64 Kernel Module  525.89.02  Wed Feb  1 23:23:25 UTC 2023
|  GCC version:  gcc version 11.4.0 (Ubuntu 11.4.0-1ubuntu1~22.04) 

|  	linux-vdso.so.1 (0x00007ffce1bf9000)
|  	libcurl.so.4 => /lib/x86_64-linux-gnu/libcurl.so.4 (0x00007f51e4364000)
|  	libz.so.1 => /lib/x86_64-linux-gnu/libz.so.1 (0x00007f51e4348000)
|  	libimf.so => /global/software/TeraChem/2025.01-intel-2023.2.0-CUDA-12.2.1/lib/libimf.so (0x00007f51e3f5e000)
|  	libm.so.6 => /lib/x86_64-linux-gnu/libm.so.6 (0x00007f51e3e75000)
|  	libmpi.so.12 => /global/software/TeraChem/2025.01-intel-2023.2.0-CUDA-12.2.1/lib/libmpi.so.12 (0x00007f51e2200000)
|  	libdftbplus.so => /global/software/TeraChem/2025.01-intel-2023.2.0-CUDA-12.2.1/lib/libdftbplus.so (0x00007f51e1ea8000)
|  	libfsockets.so => /global/software/TeraChem/2025.01-intel-2023.2.0-CUDA-12.2.1/lib/libfsockets.so (0x00007f51e3e6f000)
|  	libxmlf90.so => /global/software/TeraChem/2025.01-intel-2023.2.0-CUDA-12.2.1/lib/libxmlf90.so (0x00007f51e3e44000)
|  	libthcbox.so.1 => /global/software/TeraChem/2025.01-intel-2023.2.0-CUDA-12.2.1/lib/libthcbox.so.1 (0x00007f51e008b000)
|  	libOpenMM.so => /global/software/TeraChem/2025.01-intel-2023.2.0-CUDA-12.2.1/lib/libOpenMM.so (0x00007f51dfc31000)
|  	libFlexiBLE.so => /global/software/TeraChem/2025.01-intel-2023.2.0-CUDA-12.2.1/lib/libFlexiBLE.so (0x00007f51e3e2f000)
|  	libnvToolsExt.so.1 => /global/software/TeraChem/2025.01-intel-2023.2.0-CUDA-12.2.1/lib/libnvToolsExt.so.1 (0x00007f51dfa00000)
|  	libgpubox.so.1 => /global/software/TeraChem/2025.01-intel-2023.2.0-CUDA-12.2.1/lib/libgpubox.so.1 (0x00007f51e3e24000)
|  	libcpubox.so.1 => /global/software/TeraChem/2025.01-intel-2023.2.0-CUDA-12.2.1/lib/libcpubox.so.1 (0x00007f51e3e1e000)
|  	libtensorbox.so.1 => /global/software/TeraChem/2025.01-intel-2023.2.0-CUDA-12.2.1/lib/libtensorbox.so.1 (0x00007f51e3e00000)
|  	libiobox.so.1 => /global/software/TeraChem/2025.01-intel-2023.2.0-CUDA-12.2.1/lib/libiobox.so.1 (0x00007f51e3dfa000)
|  	libintbox.so.1 => /global/software/TeraChem/2025.01-intel-2023.2.0-CUDA-12.2.1/lib/libintbox.so.1 (0x00007f5179ea1000)
|  	libatmx2cbox.so.1 => /global/software/TeraChem/2025.01-intel-2023.2.0-CUDA-12.2.1/lib/libatmx2cbox.so.1 (0x00007f51e3d39000)
|  	libperiodicbox.so.1 => /global/software/TeraChem/2025.01-intel-2023.2.0-CUDA-12.2.1/lib/libperiodicbox.so.1 (0x00007f515f01a000)
|  	libsqmbox.so.1 => /global/software/TeraChem/2025.01-intel-2023.2.0-CUDA-12.2.1/lib/libsqmbox.so.1 (0x00007f515ec6e000)
|  	libgridbox.so.1 => /global/software/TeraChem/2025.01-intel-2023.2.0-CUDA-12.2.1/lib/libgridbox.so.1 (0x00007f515c8a4000)
|  	libdftbox.so.1 => /global/software/TeraChem/2025.01-intel-2023.2.0-CUDA-12.2.1/lib/libdftbox.so.1 (0x00007f515c7a4000)
|  	libcibox.so.1 => /global/software/TeraChem/2025.01-intel-2023.2.0-CUDA-12.2.1/lib/libcibox.so.1 (0x00007f515c155000)
|  	libccbox.so.1 => /global/software/TeraChem/2025.01-intel-2023.2.0-CUDA-12.2.1/lib/libccbox.so.1 (0x00007f515bcc4000)
|  	libsimbox.so.1 => /global/software/TeraChem/2025.01-intel-2023.2.0-CUDA-12.2.1/lib/libsimbox.so.1 (0x00007f51e3d0f000)
|  	liboptbox.so.1 => /global/software/TeraChem/2025.01-intel-2023.2.0-CUDA-12.2.1/lib/liboptbox.so.1 (0x00007f51e3cd8000)
|  	libprotobuf.so.25 => /global/software/TeraChem/2025.01-intel-2023.2.0-CUDA-12.2.1/lib/libprotobuf.so.25 (0x00007f515b9f1000)
|  	libcommbox.so.1 => /global/software/TeraChem/2025.01-intel-2023.2.0-CUDA-12.2.1/lib/libcommbox.so.1 (0x00007f515b990000)
|  	libssl.so.3 => /lib/x86_64-linux-gnu/libssl.so.3 (0x00007f515b8ec000)
|  	libcrypto.so.3 => /lib/x86_64-linux-gnu/libcrypto.so.3 (0x00007f515b4a9000)
|  	libcuda.so.1 => /lib/x86_64-linux-gnu/libcuda.so.1 (0x00007f51597c9000)
|  	libcudart.so.12 => /global/software/TeraChem/2025.01-intel-2023.2.0-CUDA-12.2.1/lib/libcudart.so.12 (0x00007f5159400000)
|  	libcublas.so.12 => /global/software/TeraChem/2025.01-intel-2023.2.0-CUDA-12.2.1/lib/libcublas.so.12 (0x00007f5152c00000)
|  	libcufft.so.11 => /global/software/TeraChem/2025.01-intel-2023.2.0-CUDA-12.2.1/lib/libcufft.so.11 (0x00007f5147e00000)
|  	libcusparse.so.12 => /global/software/TeraChem/2025.01-intel-2023.2.0-CUDA-12.2.1/lib/libcusparse.so.12 (0x00007f5138000000)
|  	libcusolver.so.11 => /global/software/TeraChem/2025.01-intel-2023.2.0-CUDA-12.2.1/lib/libcusolver.so.11 (0x00007f5130e00000)
|  	libcublasLt.so.12 => /global/software/TeraChem/2025.01-intel-2023.2.0-CUDA-12.2.1/lib/libcublasLt.so.12 (0x00007f510e600000)
|  	libsvml.so => /global/software/TeraChem/2025.01-intel-2023.2.0-CUDA-12.2.1/lib/libsvml.so (0x00007f510cfd1000)
|  	libirng.so => /global/software/TeraChem/2025.01-intel-2023.2.0-CUDA-12.2.1/lib/libirng.so (0x00007f510cc00000)
|  	libstdc++.so.6 => /lib/x86_64-linux-gnu/libstdc++.so.6 (0x00007f510c9d4000)
|  	libiomp5.so => /global/software/TeraChem/2025.01-intel-2023.2.0-CUDA-12.2.1/lib/libiomp5.so (0x00007f510c400000)
|  	libcilkrts.so.5 => /global/software/TeraChem/2025.01-intel-2023.2.0-CUDA-12.2.1/lib/libcilkrts.so.5 (0x00007f510c000000)
|  	libgcc_s.so.1 => /lib/x86_64-linux-gnu/libgcc_s.so.1 (0x00007f51dfc11000)
|  	libintlc.so.5 => /global/software/TeraChem/2025.01-intel-2023.2.0-CUDA-12.2.1/lib/libintlc.so.5 (0x00007f5159751000)
|  	libc.so.6 => /lib/x86_64-linux-gnu/libc.so.6 (0x00007f510bdd7000)
|  	/lib64/ld-linux-x86-64.so.2 (0x00007f51e441a000)
|  	libnghttp2.so.14 => /lib/x86_64-linux-gnu/libnghttp2.so.14 (0x00007f5159727000)
|  	libidn2.so.0 => /lib/x86_64-linux-gnu/libidn2.so.0 (0x00007f5159706000)
|  	librtmp.so.1 => /lib/x86_64-linux-gnu/librtmp.so.1 (0x00007f51596e7000)
|  	libssh.so.4 => /lib/x86_64-linux-gnu/libssh.so.4 (0x00007f5152b93000)
|  	libpsl.so.5 => /lib/x86_64-linux-gnu/libpsl.so.5 (0x00007f51596d3000)
|  	libgssapi_krb5.so.2 => /lib/x86_64-linux-gnu/libgssapi_krb5.so.2 (0x00007f5152b3f000)
|  	libldap-2.5.so.0 => /lib/x86_64-linux-gnu/libldap-2.5.so.0 (0x00007f5147da1000)
|  	liblber-2.5.so.0 => /lib/x86_64-linux-gnu/liblber-2.5.so.0 (0x00007f51596c2000)
|  	libzstd.so.1 => /lib/x86_64-linux-gnu/libzstd.so.1 (0x00007f5137f31000)
|  	libbrotlidec.so.1 => /lib/x86_64-linux-gnu/libbrotlidec.so.1 (0x00007f51596b4000)
|  	librt.so.1 => /lib/x86_64-linux-gnu/librt.so.1 (0x00007f51596af000)
|  	libdl.so.2 => /lib/x86_64-linux-gnu/libdl.so.2 (0x00007f51596aa000)
|  	libifport.so.5 => /global/software/TeraChem/2025.01-intel-2023.2.0-CUDA-12.2.1/lib/libifport.so.5 (0x00007f51593d6000)
|  	libifcoremt.so.5 => /global/software/TeraChem/2025.01-intel-2023.2.0-CUDA-12.2.1/lib/libifcoremt.so.5 (0x00007f510c85a000)
|  	libpthread.so.0 => /lib/x86_64-linux-gnu/libpthread.so.0 (0x00007f51593d1000)
|  	libnvJitLink.so.12 => /global/software/TeraChem/2025.01-intel-2023.2.0-CUDA-12.2.1/lib/libnvJitLink.so.12 (0x00007f5108c00000)
|  	libunistring.so.2 => /lib/x86_64-linux-gnu/libunistring.so.2 (0x00007f510c256000)
|  	libgnutls.so.30 => /lib/x86_64-linux-gnu/libgnutls.so.30 (0x00007f5108a15000)
|  	libhogweed.so.6 => /lib/x86_64-linux-gnu/libhogweed.so.6 (0x00007f5137ee9000)
|  	libnettle.so.8 => /lib/x86_64-linux-gnu/libnettle.so.8 (0x00007f5137ea3000)
|  	libgmp.so.10 => /lib/x86_64-linux-gnu/libgmp.so.10 (0x00007f5130d7e000)
|  	libkrb5.so.3 => /lib/x86_64-linux-gnu/libkrb5.so.3 (0x00007f510894a000)
|  	libk5crypto.so.3 => /lib/x86_64-linux-gnu/libk5crypto.so.3 (0x00007f5137e74000)
|  	libcom_err.so.2 => /lib/x86_64-linux-gnu/libcom_err.so.2 (0x00007f51593c9000)
|  	libkrb5support.so.0 => /lib/x86_64-linux-gnu/libkrb5support.so.0 (0x00007f5152b31000)
|  	libsasl2.so.2 => /lib/x86_64-linux-gnu/libsasl2.so.2 (0x00007f5137e59000)
|  	libbrotlicommon.so.1 => /lib/x86_64-linux-gnu/libbrotlicommon.so.1 (0x00007f5130d5b000)
|  	libp11-kit.so.0 => /lib/x86_64-linux-gnu/libp11-kit.so.0 (0x00007f510880f000)
|  	libtasn1.so.6 => /lib/x86_64-linux-gnu/libtasn1.so.6 (0x00007f5130d43000)
|  	libkeyutils.so.1 => /lib/x86_64-linux-gnu/libkeyutils.so.1 (0x00007f51593c0000)
|  	libresolv.so.2 => /lib/x86_64-linux-gnu/libresolv.so.2 (0x00007f5137e43000)
|  	libffi.so.8 => /lib/x86_64-linux-gnu/libffi.so.8 (0x00007f5130d36000)
| #################################################################################################

| Checking Amazon Cloud instance ID...   Not found
| Cannot find license.dat file /global/software/TeraChem/2025.01-intel-2023.2.0-CUDA-12.2.1/license.dat


| TRYING THE NETWORK LICENSE...
| Connecting to license server '54.208.252.40' port '8877'...
| Connected!
| Checking your license...

| **************************************************************
|  Greetings, Martinez Group! You have 1000 licenses in total
|  IN USE: 44
|  AVAILABLE: 956
| **************************************************************

-----------------------------------
Processed Input file:
                   method uhf
                   charge 1
                 spinmult 2
              coordinates wat1.xyz
                     resp yes
                      run gradient
                    basis 6-31g*
                  threall 1e-15
                 convthre 1e-6
                precision double
-----------------------------------
Jobname: wat1
Scratch data will be generated
Scratch directory: ./scr.wat1
| Random number seed: 625717612

XYZ coordinates wat1.xyz
Molden File Output: ./scr.wat1/wat1.molden
Using basis set: 6-31gs
dmrgstart not found
Spin multiplicity: 2
DIIS will use up to 10 vectors.
Condition number limit for the DIIS overlap matrix is            0
WF convergence threshold: 1.00e-06
Using DIIS algorithm to converge WF
Maximum number of SCF iterations: 100
Incremental fock with rebuild every 8 iterations
Will switch to conventional Fock if diffuse functions are detected
X-matrix tolerance: 1.00e-04
PRECISION: DOUBLE
TeraChem will select linear algebra engine
DFT Functional requested: hf
Method: Hartree-Fock 
Wavefunction: UNRESTRICTED
Initial guess generated by maximum overlap

********************************************
**** SINGLE POINT GRADIENT CALCULATIONS ****
********************************************
SCF:   Initial guess is taken from previous step
Starting TensorBox...
| TeraChem will select linear algebra engine
| Using cuSolver for matrix diagonalization

| using 1 out of 1 CUDA devices
|   Device 0: NVIDIA GeForce GTX TITAN X, 12207MB, CC 5.2 -- CPU THREAD 0
| -------------------------------------------------------------------
| CPU Memory Available: 18127.88 MegaWords 
| GPU Memory Available: 1269.88 MegaWords 
| Maximum recommended basis set size: 13300 basis functions
|  (limited by GPU memory)
| -------------------------------------------------------------------
| Initialize GPUBox with 128 MW of memory

****** QM coordinates ******
O         0.6840000000        -1.7170000000        -2.0850000000  
H         0.3970000000        -1.1190000000        -1.4460000000  
H         1.1780000000        -2.1830000000        -2.8040000000  

Basis set:       6-31gs
Total atoms:     3
Total charge:    1
Total electrons: 9 (5-alpha, 4-beta)
Number electrons modeled by ECPs: 0
Total orbitals:  19
Total AO shells: 10 (7 S-shells; 2 P-shells; 1 D-shells; 0 F-shells; 0 G-shells)
Spin multiplicity: 2
Nuclear repulsion energy (QM atoms): 9.156271155706 a.u.


                      *** Start SCF Iterations ***

 Iter     DIIS Error       Energy change         Energy        Time(s)
------------------------------------------------------------------------
| Purifying alpha guess...
| Trace of guess   5.00000000, trace of guess*guess   5.00000000,  electron number   5.00000000 
| Guess is well-conditioned: purify...
| Purification run on CPU (double precision)  using TC2 purification.
| Density matrix purification finished after a total of 0 matrix multiplies
| Purifying beta guess...
| Trace of guess   4.00000000, trace of guess*guess   4.00000000,  electron number   4.00000000 
| Guess is well-conditioned: purify...
| Purification run on CPU (double precision)  using TC2 purification.
| Density matrix purification finished after a total of 0 matrix multiplies
|    1     1.0679333602     -74.7648281801     -74.7648281801     0.09  
|    2     0.3814258860      -0.7011432786     -75.4659714586     0.04  
|    3     0.0533905417      -0.1097623773     -75.5757338359     0.04  
|    4     0.0078984450      -0.0028946830     -75.5786285189     0.04  
|    5     0.0023881525      -0.0001448292     -75.5787733481     0.04  
|    6     0.0008026336      -0.0000125481     -75.5787858962     0.04  
|    7     0.0005434382      -0.0000031432     -75.5787890394     0.04  
|    8     0.0005345790      -0.0000018643     -75.5787909037     0.03  
|    9     0.0005289400      -0.0000014445     -75.5787923483     0.03  
|   10     0.0005034415      -0.0000070098     -75.5787993581     0.03  
|   11     0.0003326439      -0.0000411034     -75.5788404614     0.04  
|   12     0.0003303358      +0.0000008089     -75.5788396525     0.03  
|   13     0.0011340409      -0.0000271639     -75.5788668164     0.03  
|   14     0.0005314322      +0.0000002551     -75.5788665613     0.03  
|   15     0.0004094301      -0.0000024192     -75.5788689805     0.03  
|   16     0.0000649330      -0.0000001520     -75.5788691325     0.03  
|   17     0.0000253676      -0.0000000042     -75.5788691367     0.03  
|   18     0.0000007583      -0.0000000005     -75.5788691372     0.03  
------------------------------------------------------------------------
FINAL ENERGY: -75.5788691372 a.u.
HOMO-LUMO GAP: alpha: 23.509 eV
               beta:  18.032 eV 
CENTER OF MASS: {0.695583, -1.709614, -2.089477} ANGS
DIPOLE MOMENT: {0.592470, 0.309352, -0.275266} (|D| = 0.722835) DEBYE
SPIN SZ:         0.500000
SPIN S-SQUARED: 0.756818 (exact: 0.75)

Computing ESP on molecular grid...
Using layered Connolly surfaces for molecular grid...Total number of ESP grid points from 4 layers: 299
Electrostatic potential written to file ./scr.wat1/esp.xyz (unit: Ansgtrom)
Fitting RESP charges to the computed ESP...
ESP unrestrained charges:
Atom          X          Y          Z     Charge   Exposure
-----------------------------------------------------------
   O   1.292573  -3.244660  -3.940079  -0.269077     0.7790
   H   0.750221  -2.114603  -2.732544   0.626746     0.5891
   H   2.226097  -4.125272  -5.298792   0.642332     0.6357
-----------------------------------------------------------
Quality of fit (RRMS):0.0187

Solving Resp charge with restraints iteratively...
Restraint parameters a:0.0005 , b:0.001 
ESP restrained charges:
Atom          X          Y          Z     Charge   Exposure
-----------------------------------------------------------
   O   1.292573  -3.244660  -3.940079  -0.260148     0.7790
   H   0.750221  -2.114603  -2.732544   0.622120     0.5891
   H   2.226097  -4.125272  -5.298792   0.638028     0.6357
-----------------------------------------------------------
Quality of fit (RRMS):0.0187


Running Mulliken population analysis...


Gradient units are Hartree/Bohr
---------------------------------------------------
        dE/dX            dE/dY            dE/dZ
   -0.0006380621     0.0701256950     0.0479355141
    0.0089523415    -0.0561717083    -0.0452998935
   -0.0083142794    -0.0139539867    -0.0026356206
---------------------------------------------------
Net gradient: -2.270753e-15  1.162265e-16  1.991463e-15
Net torque:   -6.272942e-07 -1.057772e-06  1.066426e-07

Writing out molden info
| Total processing time: 1.01 sec 

| Job finished: Wed Feb 19 10:43:42 2025

